subsys/net/l2/openthread/Kconfig.thread

```diff
@@ -24,6 +24,12 @@ config OPENTHREAD_XPANID
 	  Extended PAN ID for OpenThread with
 	  format "de:ad:00:be:ef:00:ca:fe"
 
+config OPENTHREAD_MASTERKEY
```

Comment:
Aren't we missing new options from https://github.com/zephyrproject-rtos/openthread/pull/35/commits/28df62ec3e46471ed562cbf4a05d6dac063e2aa9 here? I.e. `CONFIG_OPENTHREAD_SRP_CLIENT` and `CONFIG_OPENTHREAD_SRP_SERVER`

Reply:
I was sure to have added those. Will update, thanks.